react/Checkbox/Checkbox.less

```diff
@@ -34,14 +37,20 @@
   @media @desktop {
     .touchableText(@standard-type-scale);
   }
+
+  &.isMultiline {
+    .standardTextResponsive(@baseline: false);
```

Comment:
We shouldn't be adding more of usages these mixins, they've been deprecated in favour of the `<Text>` component.